scripts/build/Jenkins/Jenkinsfile

```diff
@@ -823,6 +823,37 @@ def UploadCrashArtifacts(String platform) {
     }
 }
 
+def UploadBuildArtifacts(String workspace, String s3sisUploadParams = "") {
+    catchError(message: "Error uploading build artifacts (this won't fail the build)", buildResult: 'UNSTABLE', stageResult: 'FAILURE') {
+        dir(workspace) {
+            checkout scm: [
+                $class: 'GitSCM',
+                branches: [[name: '*/configure']],
+                extensions: [
+                    [$class: 'AuthorInChangelog'],
+                    [$class: 'RelativeTargetDirectory', relativeTargetDir: 's3sis']
+                ],
+                userRemoteConfigs: [[url: "https://github.com/aws-lumberyard/s3sis.git", credentialsId: "${env.GITHUB_USER}"]]
```

Comment:
We may need to host the s3sis installer in another location since this repo is supposed to be private. I'll reach out offline for details.